omas/omas_plot.py

```diff
@@ -685,6 +687,8 @@ def equilibrium_CX(
     label = kw.pop('label', '')
     kw1 = copy.deepcopy(kw)
     kw1['linewidth'] = kw['linewidth'] + 1
+    for contour_only_keyword in ['linestyles', 'linewidths', 'colors']:
+        kw1.pop(contour_only_keyword, None)  # Remove keywords that only affect contours so they don't offend plot
```

Comment:
This probably should've been separate, since it's unrelated, but it's nice because it allows plot keywords for the contours to be used without crashing the boundary outline plot.